src/Databricks/test/utils.ps1

```diff
@@ -8,7 +8,7 @@ function RandomString([bool]$allChars, [int32]$len) {
 $env = @{}
 function setupEnv() {
     Write-Host -ForegroundColor Yellow "WARNING: Need to use Az.KeyVault module, Please check if installed Az.KeyVault(2.0.0 or Greater)."
-    # Import-Module -Name Az.KeyVault
+    Import-Module -Name Az.KeyVault
```

Comment:
Should this line be commented out?

Reply:
I don't think it should be commented out because it will not execute when the test case is run through the playback model.